src/components/App/App.js

```diff
@@ -1,26 +1,26 @@
-import React from 'react';
-import logo from './logo.svg';
+import  React, {Component} from 'react';
 import './App.css';
+import Login from '../Login/Login'
+import AreaContainer from '../AreaContainer/AreaContainer'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+// import { render } from '@testing-library/react';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      current: 'login'
```

Comment:
Lesson tomorrow on React Router - I think this should help us a lot in terms of when to render certain pages and how to make sure the URL responds accordingly.